packages/manager/modules/pci/src/projects/project/storages/containers/add/add.html

```diff
@@ -1,7 +1,28 @@
-<h1
-    data-translate="pci_projects_project_storages_containers_add_title"
-    data-ng-if="!$ctrl.archive"
-></h1>
+<div data-ng-if="!$ctrl.archive">
+    <h1
+        class="mb-1"
+        data-translate="pci_projects_project_storages_containers_add_title"
+    ></h1>
+    <h2 class="oui-header__description mb-5">
+        <div
+            data-translate="pci_projects_project_storages_containers_add_description"
+        ></div>
+        <a
+            href="{{ $ctrl.STORAGE_PRICES_LINK }}"
+            class="oui-link_icon"
+            rel="noopener noreferrer"
```

Comment:
`noreferrer` implicitly makes the browser behaves as if there was the `noopener` option.
```suggestion
            rel="noreferrer"
```

Reply:
:+1: